### PR TITLE
Bankrate author

### PR DESCRIPTION
**Pull request by toniQ97**
# Description

Bankrate author styling

# Screenshots

### Desktop 
image
### Mobile
image

# Checklist

Pull request contains:

- [ ] A new component
- [X] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [X] Work has been tested in multiple browsers
- [X] PR description includes description of change
- [X] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/themes/bankrate/theme.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
   },
   "lineHeights": {
     "base": "1.5",
-    "heading": "1.3"
+    "heading": "1.3",
+    "sm": "16px",
```
Comment (StefanChristou) [inline src/themes/bankrate/theme.json:27]: It's better to use the spacing variables defined in the top of the file rather than hard-coded values

Comment (toniQ97) [reply src/themes/bankrate/theme.json:27]: I know, but design requires 16px, and Chris Underdown explicitly said it is required, 1.3 turns out to be more than 16px

Comment (StefanChristou) [reply src/themes/bankrate/theme.json:27]: All good

**Review by StefanChristou — APPROVED** [review]
Nice work. Thanks!

**Review by StefanChristou — APPROVED** [review]

**Pull request merged**